projects/1.20-fabric/assets/tridents-n-stuff/tridents_n_stuff/lang/zh_cn.json

```
@@ -0,0 +1,79 @@
+{
+  "entity.tridents_n_stuff.slumbering_trident_entity": "沉睡的三叉戟",
+  "entity.tridents_n_stuff.stormseeker_entity": "风暴追觅者",
+  "entity.tridents_n_stuff.the_desecrator_entity": "亵渎者",
+  "entity.tridents_n_stuff.echo_of_the_deep_entity": "深渊回响",
+  "entity.tridents_n_stuff.stellaris_entity": "群星",
+  "entity.tridents_n_stuff.sanguine_bond_entity": "猩红契约",
+  "entity.tridents_n_stuff.frenzied_trident_entity": "癫狂三叉戟",
+  "entity.tridents_n_stuff.ancient_trident_entity": "上古三叉戟",
+  "entity.tridents_n_stuff.oceanic_trident_entity": "深海三叉戟",
+  "entity.tridents_n_stuff.holy_trident_entity": "神圣三叉戟",
+  "entity.tridents_n_stuff.farshot_trident_entity": "远射三叉戟",
+  "entity.tridents_n_stuff.bone_harpoon_entity": "骨鱼叉",
+  "entity.tridents_n_stuff.prismarine_harpoon_entity": "海晶石鱼叉",
+  "entity.tridents_n_stuff.diamond_harpoon_entity": "钻石鱼叉",
+  "entity.tridents_n_stuff.wooden_spear_entity": "木矛",
+  "entity.tridents_n_stuff.stone_spear_entity": "石矛",
+  "entity.tridents_n_stuff.iron_spear_entity": "铁矛",
+  "entity.tridents_n_stuff.golden_spear_entity": "金矛",
+  "entity.tridents_n_stuff.diamond_spear_entity": "钻石矛",
+  "entity.tridents_n_stuff.netherite_spear_entity": "下界合金矛",
+  "item.tridents_n_stuff.slumbering_trident": "沉睡的三叉戟",
+  "item.tridents_n_stuff.slumbering_trident.flavor": "古老的力量在其中沉睡...一定有什么东西能唤醒它",
+  "item.tridents_n_stuff.stormseeker": "风暴追觅者",
+  "item.tridents_n_stuff.stormseeker.flavor": "来自雷暴中心的狂怒充斥其中",
+  "item.tridents_n_stuff.the_desecrator": "亵渎者",
+  "item.tridents_n_stuff.the_desecrator.flavor": "众神的鲜血从它污秽不堪的尖刺上滴落",
+  "item.tridents_n_stuff.echo_of_the_deep": "深渊回响",
+  "item.tridents_n_stuff.echo_of_the_deep.flavor": "在它叉齿间回荡的鲸鱼歌声里，还混杂着若干不可名状的骇人声响",
+  "item.tridents_n_stuff.stellaris": "群星",
+  "item.tridents_n_stuff.stellaris.flavor": "握在手里像星辰般沉重且冰冷。它似乎想要把周遭的一切都吸进体内",
+  "item.tridents_n_stuff.sanguine_bond": "猩红契约",
+  "item.tridents_n_stuff.sanguine_bond.flavor": "它猩红色的表面似乎渴望着鲜血",
+  "item.tridents_n_stuff.icon_of_storms": "风暴刻印",
+  "item.tridents_n_stuff.icon_of_storms.flavor": "唤醒风暴追觅者",
+  "item.tridents_n_stuff.icon_of_evil": "邪恶刻印",
+  "item.tridents_n_stuff.icon_of_evil.flavor": "唤醒亵渎者",
+  "item.tridents_n_stuff.icon_of_depths": "深渊刻印",
+  "item.tridents_n_stuff.icon_of_depths.flavor": "唤醒深渊回响",
+  "item.tridents_n_stuff.icon_of_stars": "星辰刻印",
+  "item.tridents_n_stuff.icon_of_stars.flavor": "唤醒群星",

```

```suggestion
  "item.tridents_n_stuff.icon_of_stars.flavor": "贮藏着群星的能量",
```
原文`Хранит энергию звёзд`